x-pack/plugins/observability/public/embeddable/slo/alerts/slo_alerts_wrapper.tsx

```diff
@@ -0,0 +1,127 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0; you may not use this file except in compliance with the Elastic License
+ * 2.0.
+ */
+import React, { useState, useEffect } from 'react';
+import { FormattedMessage } from '@kbn/i18n-react';
+import { i18n } from '@kbn/i18n';
+import { EuiFlexGroup, EuiFlexItem, EuiLink } from '@elastic/eui';
+import type { TimeRange } from '@kbn/es-query';
+import { CONTEXT_MENU_TRIGGER } from '@kbn/embeddable-plugin/public';
+import { IEmbeddable, EmbeddableOutput } from '@kbn/embeddable-plugin/public';
+import { ActionExecutionContext } from '@kbn/ui-actions-plugin/public';
+import { SloAlertsSummary } from './components/slo_alerts_summary';
+import { SloAlertsTable } from './components/slo_alerts_table';
+import type { SloItem } from './types';
+import { SloEmbeddableDeps } from './slo_alerts_embeddable';
+import { SloAlertsEmbeddableInput } from './types';
+import { EDIT_SLO_ALERTS_ACTION } from '../../../ui_actions/edit_slo_alerts_panel';
+import { paths } from '../../../../common/locators/paths';
+
+interface Props {
+  deps: SloEmbeddableDeps;
+  slos: SloItem[];
+  timeRange: TimeRange;
+  embeddable: IEmbeddable<SloAlertsEmbeddableInput, EmbeddableOutput>;
+  onRenderComplete?: () => void;
+}
+
+export function SloAlertsWrapper({ embeddable, slos, deps, timeRange, onRenderComplete }: Props) {
+  const {
+    application: { navigateToUrl },
+    http: { basePath },
+  } = deps;
+
+  const [isSummaryLoaded, setIsSummaryLoaded] = useState(false);
+  const [isTableLoaded, setIsTableLoaded] = useState(false);
+  useEffect(() => {
+    if (!onRenderComplete) {
+      return;
+    }
+    if (isSummaryLoaded && isTableLoaded) {
+      onRenderComplete();
+    }
+  }, [isSummaryLoaded, isTableLoaded, onRenderComplete]);
+  const handleGoToAlertsClick = () => {
+    let kuery = '';
+    slos.map((slo, index) => {
+      const shouldAddOr = index < slos.length - 1;
+      kuery += `slo.id:"${slo.id}" and slo.instanceId:"${slo.instanceId}"`;
+      if (shouldAddOr) {
+        kuery += ' or ';
+      }
+    });
+    navigateToUrl(
+      `${basePath.prepend(paths.observability.alerts)}?_a=(kuery:'${kuery}',rangeFrom:${
+        timeRange.from
+      },rangeTo:${timeRange.to})`
+    );
+  };
+
+  return (
+    <>
```

Comment:
nit -> all these styles attributes should be declare as const to avoid re-rendering

Reply:
@mgiota  We should do this